classic/src/view/form/Print.js

```diff
@@ -41,135 +59,32 @@ Ext.define("MoMo.client.view.form.Print", {
         this.provider = Ext.create('GeoExt.data.MapfishPrintProvider', {
             url: this.getUrl() + appCombo.getValue() + '/capabilities.json',
             listeners: {
-                ready: 'onPrintProviderReady',
+                ready: function(){
+                    this.getController().onPrintProviderReady(this.provider);
```

Comment:
I need to submit a argument to the function (in this case `this.provider`). If you know the way, how I can do it otherwise, I would be glad to learn it :smile_cat: